### PR TITLE
fix(layers): support binary geometry on WebGPU

### PR DESCRIPTION
**Pull request by ibgreen-openai**
## Goal

Prepare `ScatterplotLayer` and `PathLayer` to render binary geometry reliably on WebGPU. These fixes are prerequisites for enabling binary `MVTLayer` rendering, but do not add clipping support themselves.

## Changes

- Pack ScatterplotLayer's scalar/color instance attributes into one WebGPU buffer group so binary points remain within WebGPU's vertex-buffer limit.
- Rename PathLayer's packed WebGPU parent attribute so it is not mistaken for an anchor-position attribute.
- Build PathLayer's WebGPU neighbor window directly from external binary `getPath` data, respecting XY/XYZ size, byte stride, and byte offset.
- Add a focused WebGPU binary GeoJSON regression test covering point and path pipeline validation and non-empty packed path positions.

## Validation

- `npx vitest run --project headless test/modules/layers/geojson-layer-webgpu.spec.ts` — passed
- commit hook node tests — 15 passed
- `yarn lint` — passed
- `git diff --check` — passed
- `yarn build` — WebGL-only and WebGPU TypeScript/declaration/package compilation passed; the final unrelated `lerna run build` wrapper failed to construct its Nx project graph in the isolated worktree

### CODE REVIEW

File: modules/layers/src/path-layer/path-layer.ts [hunk 1]
```diff
@@ -433,9 +434,22 @@ export default class PathLayer<DataT = any, ExtraPropsT extends {} = {}> extends
...
   protected calculateWebGPUPositions(attribute) {
     const {pathTesselator} = this.state;
-    const value = pathTesselator.get('positions');
+    const binaryPositions = (this.props.data as any)?.attributes?.getPath;
+    let value = binaryPositions ? binaryPositions.value || binaryPositions : null;
+    let sourceSize = 3;
+    let sourceStride = 3;
+    let sourceOffset = 0;
+
+    if (binaryPositions && ArrayBuffer.isView(value)) {
```
Comment (Pessimistress) [inline modules/layers/src/path-layer/path-layer.ts:443]: Binary attribute is handled in the Tesselator.
